iroha_introspect/iroha_introspect_bin/src/main.rs

```diff
@@ -0,0 +1,34 @@
+//! Binary to print all types to json string
+
+#![allow(clippy::print_stdout)]
+
+use std::collections::BTreeMap;
+
+use iroha_data_model::prelude::*;
+use iroha_data_model::query::QueryBox;
+use iroha_introspect::prelude::*;
+
+macro_rules! set_types {
```

Comment:
The naming of the macro `set_types` could be improved - try `to_json` or something similar.